js-sdkTests/signTxn.js

```diff
@@ -1,27 +1,17 @@
-import LitJsSdk from "lit-js-sdk/build/index.node.js";
-import fs from "fs";
+import * as LitJsSdk from "@lit-protocol/lit-node-client";
 import { serialize, recoverAddress } from "@ethersproject/transactions";
 import {
-  hexlify,
   splitSignature,
-  hexZeroPad,
   joinSignature,
+  arrayify
 } from "@ethersproject/bytes";
 import { recoverPublicKey, computePublicKey } from "@ethersproject/signing-key";
 import { ethers } from "ethers";
 
 // this code will be run on the node
-const litActionCode = fs.readFileSync("./build/signTxnTest.js");
-
-// you need an AuthSig to auth with the nodes
-// normally you would obtain an AuthSig by calling LitJsSdk.checkAndSignAuthMessage({chain})
-// NOTE: to replace with a new one that you get from oauth-pkp-signup-example
-const authSig = {
-  "sig": "0xe5e1d290ed645fd45d558dc8de176fc808dffb4bdc1c1c24e1cdc15f38f30a72251ef9c1e48bfd95568818d97894aff5ae1e2cc9da30c317e54262074f21823b1b",
-  "derivedVia": "web3.eth.personal.sign via Lit PKP",
-  "signedMessage": "localhost:3000 wants you to sign in with your Ethereum account:\n0x2a5A2A9558118388e8f4bd1e1c32ac520CA7D0F4\n\nLit Protocol PKP session signature\n\nURI: lit:session:9e0525c8caa0e70f85b829677f5abab27ae70344679ab67d9eda90c10b3160e2\nVersion: 1\nChain ID: 1\nNonce: 9OtojzBuln2qoTiH2\nIssued At: 2022-12-26T00:46:24.514Z\nExpiration Time: 2024-12-25T00:46:23.433Z\nResources:\n- urn:recap:lit:session:eyJkZWYiOlsibGl0RW5jcnlwdGlvbkNvbmRpdGlvbiJdfQ==",
-  "address": "0x2a5A2A9558118388e8f4bd1e1c32ac520CA7D0F4"
-};
+// cid for the bundled `build/singTxnTest.js`
+// to update run `yarn build`, upload to ipfs, and replace this cid.
+let ipfsCID = "Qmf9kGzSrNSki5DvgWs4G8FFyHMKhK7jJqHRptGosQo9NJ";
```

Comment:
why are we replacing the JS in the file with an IPFS CID?  the purpose of this repo is to demonstrate how it all works, and we have specific tests for IPFS CIDs.  I think we should keep the JS in this test.

Reply:
I think there is a issue with encoding large strings of source code for action execution, and @joshLong145 will fix it, in the interim, this is a workaround for the test to pass.

Reply:
There is an issue with serializing the provided source to base64 through the `uint8array` package's implementation which causes an `maximum call stack exceeded ` error.  The  the root cause is identified, this is to exercises the signing portion of the tests.

Reply:
with the fix to string encoding in the js-sdk [here](https://github.com/LIT-Protocol/js-sdk/commit/0cf5b2310be42a81f9fd247fa702460fe3c619c0) we are now no longer using a cid for testing